### PR TITLE
feat: Improve connext relayer

### PR DESCRIPTION
**Pull request by liu-zhipeng**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/utils/src/peripherals/gelato.ts [hunk 1]
```diff
@@ -100,36 +99,8 @@ export const getConversionRate = async (_chainId: number, to?: string, logger?:
...
-            undefined,
-            undefined,
-            jsonifyError(error as Error),
-          );
-      }
-    }, retryInterval);
-  });
   try {
-    const res = await axiosGet(apiEndpoint);
+    const res = await axiosGet(apiEndpoint, undefined, 5, 2000);
```
Comment (preethamr) [inline packages/utils/src/peripherals/gelato.ts:103]: Neat !

File: ops/testnet/prod/core/main.tf [hunk 2]
```diff
@@ -462,7 +462,36 @@ module "relayer" {
   ingress_ipv6_cdir_blocks = []
   service_security_groups  = flatten([module.network.allow_all_sg, module.network.ecs_task_sg])
   cert_arn                 = var.certificate_arn_testnet
-  container_env_vars       = local.relayer_env_vars
+  container_env_vars       = merge(local.relayer_env_vars, { RELAYER_SERVICE = "polling" })
```
Comment (preethamr) [inline ops/testnet/prod/core/main.tf:465]: This module will not have http server and, will not be able to respond to ping.

**Review by preethamr — CHANGES_REQUESTED** [review]
Not sure how this improves performance of the relayer.
Currently the relayer is not bottlenecked on http requests, it is instead on the rate at which it can place new transactions on chain.

Both https requests, and cache poll pending, happen in separate threads, already right ?

**Comment by liu-zhipeng** [comment]
> Not sure how this improves performance of the relayer. Currently the relayer is not bottlenecked on http requests, it is instead on the rate at which it can place new transactions on chain.
> 
> Both https requests, and cache poll pending, happen in separate threads, already right ?

For now, that is not bottlenecked, but when a lot of transactions are pending, so HTTP server are processing them, polling didn't process them as I tested.

File: ops/testnet/prod/core/main.tf (hunk 2 above)
Comment (liu-zhipeng) [reply ops/testnet/prod/core/main.tf:465]: could you update that?

**Review by preethamr — APPROVED** [review]

**Review by wanglonghong — APPROVED** [review]

**Pull request merged**